### PR TITLE
Add ConvertToAnonymousRecord quick fixeroony

### PR DESCRIPTION
**Pull request by cartermp**
A way to ease https://github.com/dotnet/fsharp/issues/8127

And also, in general, help people who forget the extra `|` when using anonymous records

image

### CODE REVIEW

**Comment by cartermp** [comment]
Note: there is no quick fix for opening a module or namespace that has a record with this "shape". I can try to do that in a separate PR, but it is definitely out of scope of this quick fix. If that gets implemented, it is not exclusive to this quick fix. Both would be suggestions.

File: src/fsharp/service/ServiceUntypedParse.fsi [hunk 1]
```diff
@@ -19,6 +19,9 @@ type public FSharpParseFileResults =
     /// The syntax tree resulting from the parse
     member ParseTree : ParsedInput option
 
+    /// doot doot toot toot
```
Comment (KevinRansom) [inline src/fsharp/service/ServiceUntypedParse.fsi:22]: probably a proper comment

**Review by KevinRansom — APPROVED** [review]
Fix the comment then you are good to go.

File: src/fsharp/service/ServiceUntypedParse.fsi (hunk 1 above)
Comment (cartermp) [reply src/fsharp/service/ServiceUntypedParse.fsi:22]: lol whoops

**Pull request merged**